.circleci/config.yml

```diff
@@ -903,6 +907,9 @@ workflows:
       - test-wasm2js1:
           requires:
             - build-linux
+      - test-core_2gb:
+          requires:
+            - build-linux
```

Comment:
I am a little unsure this is worth it. It won't slow the time to get the final results, but it does add to the total budget of work.

Isn't this option mostly useful for us during development? I was imagining we'd run it locally.

Reply:
I would worry that we could then be in danger of introducing regressions, but on the other hand, we do have many test modes that we don't run in CI. 

I will remove for now, once I see it passing.

Reply:
> Another option would be to run it on the testsuite bot on Chromium CI.

Yes, lets do that.